django/santropolFeast/meal/models.py

```diff
@@ -21,10 +21,10 @@
     ('veggies and fruits', _('Veggies and fruits')),
```

Comment:
@erozqba **('veggies and fruits', _('Veggies and fruits')),** should be **('veggies_and_fruits', _('Veggies and fruits')),**